src/main/java/net/dries007/tfc/common/container/ItemStackContainer.java

```diff
@@ -18,6 +20,8 @@
  */
 public class ItemStackContainer extends Container
 {
+    private static final List<ClickType> ILLEGAL_ITEM_CLICKS = List.of(ClickType.QUICK_MOVE, ClickType.PICKUP, ClickType.THROW, ClickType.SWAP);
```

Comment:
If you are only going to do `.contains`, technically a `Set` is more efficient here. If `ClickType` is an `enum`, a `EnumSet` would also work

Reply:
For future reference: If its just a tiny `List`, the performance of a `List#contains` matches or can even beat out a `Set#contains`. In this case `EnumSet` is the best option since `EnumSet#contains` just checks `(bit_vector & (1 << enum_ordinality)) != 0`